src/components/general/ButtonLayout.js

```diff
@@ -0,0 +1,142 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { View, Dimensions } from "react-native";
+
+const dimension = Dimensions.get("window");
+
+class ButtonLayout extends Component {
+	constructor(props) {
+		super(props);
+	}
+
+	static propTypes = {
+		icon: PropTypes.shape({})
+	}
+
+	render() {
+		const {
+			buttonContainer,
+			doubleLayer
+		} = styles;
+
+		const {
+			icon,
+			children,
+			containerStyle
+		} = this.props;
+
+		let i = 0;
```

Comment:
What is `i`? Can it have a more descriptive name? If it's just for looping, declare it inside the loop definition.